### PR TITLE
Start FloSeguin file

### PR DESCRIPTION
**Pull request by FloSeguin**
### Description of Change

- creating a file

### Test results

no testing

### Who should know about the change?

/cc: @githubteacher

### CODE REVIEW

File: FloSeguin.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# to do
+# places to see
```
Comment (FloSeguin) [inline FloSeguin.md:2]: Quiberon, Carnac first

File: FloSeguin.md [hunk 2]
```diff
@@ -0,0 +1,2 @@
+# to do
```
Comment (FloSeguin) [inline FloSeguin.md:1]: book flight

**Comment by CatherineMallinger** [comment]
This year we go to Normandy....

**Comment by SteveBaston** [comment]
we all need a rest

File: FloSeguin.md (hunk 1 above)
Comment (SteveBaston) [reply FloSeguin.md:2]: approved

**Review by SteveBaston — APPROVED** [review]

**Pull request merged**